pom.xml

```diff
@@ -1170,19 +1175,63 @@ under the License.
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>2.27.1</version>
+          <configuration>
+            <java>
+              <importOrder>
+                <file>config/maven-eclipse-importorder.txt</file>
+              </importOrder>
+              <removeUnusedImports/>
+              <palantirJavaFormat/>
+              <licenseHeader>
+                <file>config/maven-header-plain.txt</file>
+              </licenseHeader>
+            </java>
+            <pom>
+              <sortPom>
+                <expandEmptyElements>false</expandEmptyElements>
```

Comment:
Expanding empty elements would allow less diff noise.

Reply:
I don't see how.  This just replaces `<foo />` with `<foo></foo>` or vice-versa.  I do find the latter a bit weird to read.  Not sure how that changes diffs.  Can you elaborate ?

Reply:
Depends. If it doesn't expect child elements but just a string it's fine. If it expects child elements, the diff could have been shorter, but I really don't mind this setting. Ok for me.